Sources/Fuzzilli/Lifting/FuzzILLifter.swift

```diff
@@ -233,17 +233,25 @@ public class FuzzILLifter: Lifter {
             w.decreaseIndentionLevel()
             w.emit("EndIf")
 
-        case is BeginSwitch:
+        case let op as BeginSwitch:
```

Comment:
Thinking about this, I think I'd prefer if this was just something like
```swift
case is BeginSwitch:
    w.emit("BeginSwitch \(input(0))\(op.firstCaseIsDefaultCase ? "" : input(1))")
    w.increaseIndentionLevel()
```

Similar to how e.g. `BeginClassDefinition` is lifted. That way, the textual representation of FuzzIL stays as close to the "real" representation as possible. Otherwise, it might seem like there are multiple instructions, when in reality, it's just one.